mllib/src/main/scala/org/apache/spark/ml/ann/Layer.scala

```diff
@@ -463,7 +479,7 @@ private[ml] class FeedForwardModel private(
   private var outputs: Array[BDM[Double]] = null
   private var deltas: Array[BDM[Double]] = null
 
-  override def forward(data: BDM[Double]): Array[BDM[Double]] = {
+  override def forward(data: BDM[Double], containsLastLayer: Boolean): Array[BDM[Double]] = {
```

Comment:
Could we use a variable name like`includeLastLayer` here? `containsLastLayer` sounds like a property of the model instead of an instruction to the method.